huggan/pytorch/huggan_mixin.py

```diff
@@ -0,0 +1,124 @@
+from pathlib import Path
+from re import TEMPLATE
+from typing import Optional, Union
+
+from huggingface_hub import PyTorchModelHubMixin, HfFolder, Repository
+
+from huggan import TEMPLATE_MODEL_CARD_PATH
+
+
+class HugGANModelHubMixin(PyTorchModelHubMixin):
```

Comment:
Maybe we can add a docstring here explained that the sole purpose of overriding this class is because we want to have an automated model card creation.

Reply:
done!